playwright/plugins/docker/index.ts

```diff
@@ -74,6 +75,16 @@ export class Docker {
             }
         }
 
+        if (isPodman) {
+            // Make host.containers.internal work to allow the container to talk to other services via host ports.
+            params.push("--network");
+            params.push("slirp4netns:allow_host_loopback=true");
+        } else {
+            // Make host.docker.internal work to allow the container to talk to other services via host ports.
+            params.push("--add-host");
+            params.push("host.containers.internal:host-gateway");
```

Comment:
might be a stupid question, but how does adding `host.containers.internal` as a host affect `host.docker.internal`?

Reply:
ok, so... 

is the code not doing what the comment says? or is there some magic that I'm not following?

Reply:
Ah the comment is just outdated, needs to be updated to `.containers.` - my bad